src/coreclr/md/enc/CMakeLists.txt

```diff
@@ -33,13 +34,23 @@ set(MDRUNTIMERW_HEADERS
   ../inc/portablepdbmdds.h
   ../inc/portablepdbmdi.h
   ../inc/rwutil.h
+  ../inc/sha256.h
   ../inc/stgio.h
   ../inc/stgtiggerstorage.h
   ../inc/stgtiggerstream.h
   ../inc/streamutil.h
   ../runtime/mdinternalro.h
 )
 
+if(CLR_CMAKE_TARGET_LINUX)
+  configure_file(
+    ../../../native/libs/System.Security.Cryptography.Native/pal_crypto_config.h.in
+    ${CMAKE_CURRENT_BINARY_DIR}/pal_crypto_config.h)
+  include_directories(../../../native/libs/Common)
+  include_directories(../../../native/libs/System.Security.Cryptography.Native)
+  include_directories(${CMAKE_CURRENT_BINARY_DIR})
+endif(CLR_CMAKE_TARGET_LINUX)
+
```

Comment:
These paths look fragile... I'm interested in suggestions for linking in the OpenSSL shims more cleanly.